Lib/typing.py

```diff
@@ -1127,7 +1127,7 @@ def __call__(self, *args, **kwargs):
         result = self.__origin__(*args, **kwargs)
         try:
             result.__orig_class__ = self
-        except AttributeError:
+        except Exception:
```

Comment:
A comment here, stating that we're intentionally catching most exceptions, might be good:

```suggestion
        # Some objects raise TypeError (or something even more exotic)
        # if you try to set attributes on them; guard against that here:
        except Exception:
```

Reply:
addressed in ff5e88566e30ae887721e55829a8a3547617e510